app/styles/custom-utilities/_text.scss

```diff
@@ -17,3 +17,35 @@
 .vlc-u-font-weight-normal {
   font-weight: normal;
 }
+
+.vlc-u-font-weight-500 {
+  font-weight: 500;
+}
+
+.vlc-u-font-color-dove-gray {
+  color: $dove-gray;
```

Comment:
Would not refer to the color-names. Substitutes are better. This will be easier doing this more specifically on the component css rather than a util